src/omnisharp/protocol.ts

```diff
@@ -417,6 +417,17 @@ export interface PackageDependency {
     Name: string;
     Version: string;
 }
+    
+export interface FilesChangedRequest extends Request{
+    ChangeType: FileChangeType
```

Comment:
You called this `FileChangeType` in your `omnisharp-roslyn` PR. So this probably doesn't actually work. :smile:

Reply:
Updated the member name in the other PR.